index.html

```diff
@@ -37,9 +43,235 @@ <h1>Donice Ubaru</h1>
+              <li>Javascript</li>
+            </ul>
+          </div>
+          <button class="project-button">
+            See this project
+            <span class="material-symbols-outlined">
+              arrow_right_alt
+            </span>
+          </button>
+        </div>
+      </div>
+
+      <div>
+        <img src="./images/project-image-5.svg" alt="project-image-1">
+        <div class="project-details">
+          <h4>Project name goes here</h4>
+          <div class="project-skill">
+            <ul>
+              <li>HTML/CSS</li>
+              <li>Ruby on Rails</li>
+              <li>Javascript</li>
+            </ul>
+          </div>
+          <button class="project-button">
+            See this project
+            <span class="material-symbols-outlined">
+              arrow_right_alt
+            </span>
+          </button>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section>
+
+    <h2>About</h2>
+    <div id="About-myself">
+      <p>
+        I can help you build a product , feature or website Look through some of my work and experience! If you like
+        what
+        you see and have a project you need coded, don’t hesitate to contact me.
+      </p>
+      <button type="button">
+        Get my CV
+        <span class="material-symbols-outlined">download</span>
+      </button>
+
+      <div class="skill">
+        <div class="skill-card" id="language">
+          <h4>Languages</h4>
+          <ul>
+            <li>Javascript(ES5/ES6)</li>
+            <li>Ruby</li>
+            <li>HTML</li>
+            <li>CSS</li>
+          </ul>
+        </div>
+
+        <div class="skill-card" id="frameworks">
+          <h4>Frameworks</h4>
+          <ul>
+            <li>React.js</li>
+            <li>Ruby on rails</li>
+            <li>RSpec</li>
+            <li>Capibara</li>
+          </ul>
+        </div>
+
+        <div class="skill-card" id="skills">
+          <h4>Skills</h4>
+          <ul>
+            <li>Database Management</li>
+            <li>CLI</li>
+            <li>API Design</li>
+            <li>Version Control</li>
+            <li>Web Development</li>
+          </ul>
+        </div>
+
+      </div>
+    </div>
+  </section>
+
+
+
+  <section><img src="./images/SideImage.svg" id="nav-image" alt="" />
+    <div id="contact-me">
+
+      <h1>Get Started</h1>
+      <p>
+        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+      </p>
+
+      <form action="/">
+        <div class="form">
+          <label for="name">Your name</label><br>
+          <input id="name" type="text" placeholder="Donice Ubaru">
+          <label for="email">Your email</label><br>
+          <input id="email" type="text" placeholder="doniceubaru1@gmail.com">
+          <label for="message">Your mesaage</label><br>
+          <textarea id="message" rows="4" cols="50" placeholder="Enter yout new message here"></textarea>
```

Comment:
- [ ] KIndly remove the textarea resize icon, as it is not in the original design, check [here](https://www.w3docs.com/snippets/css/how-to-disable-the-resizing-of-the-textarea-element.html) for how

- Your Design
<img width="147" alt="image" src="https://github.com/donice/Portfolio-Microverse/assets/41163571/28fbe00c-3a5f-4dd3-93f1-b654b9879ee2">

- Figma design
<img width="139" alt="image" src="https://github.com/donice/Portfolio-Microverse/assets/41163571/86487677-2d79-4bbc-935b-3340f125ff2d">